driver/conv_driver.hpp

```diff
@@ -1175,8 +1185,8 @@ int ConvDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
             for(int i = 0; i < in_sz; i++)
             {
                 if(is_fwd || is_wrw)
-                    in.data[i] =
-                        Data_scale * RAN_GEN<Tgpu>(static_cast<Tgpu>(0.0), static_cast<Tgpu>(1.0));
+                    in.data[i] = static_cast<Tgpu>(RAN_GEN<int>(static_cast<int>(-10), static_cast<Tgpu>(10)));
```

Comment:
Why this change is necessary?

Reply:
Oh sorry, this should not changed. This is changed while I'm debugging the code. I will change it back. my fault.